tests/avocado/avocado_qemu/test.py

```diff
@@ -245,6 +252,9 @@ def get_console(self, console_address=None, prompt=r"[\#\$] "):
         :param prompt: The regex to identify we reached the prompt.
         """
 
+        if not all((self.username, self.password)):
+            raise QEMUConsoleError('Username or password not set.')
```

Comment:
I have the feeling that, for QEMU developers, a "console error" is probably going to sound like a lower level console error (about the console device itself).  This looks to me more like a `QEMULoginError`... what do you think?